### ISSUE TITLE
How to turn the decorator color to yellow

### ISSUE THREAD

**Issue by shanquan954** [eenaree/webstorm-new-ui-theme#12]
In Webstorm the decorator color should be yellow, maybe the typescript uses a function to define the decorator, so the decorator just only @ symbol is yellow, so how to fix it.by the way, at the moment of opening the file it was yellow, and then turn to blue
![example](https://github.com/eenaree/webstorm-new-ui-theme/assets/54729514/194c32ab-a3a6-4846-8643-1f0cd3c471b4)

**Comment by eenaree** [comment]
@shanquan954 

The change from yellow to blue is due to VSCode overriding the semantic token colors defined in the theme. 
I already added color definitions for decorator functions at https://github.com/eenaree/webstorm-new-ui-theme/commit/c6b85bf9bd4b416996124764e55a7b4e715d98b3.

But unfortunately, when I set `semanticHighlighting: true`, the semantic token highlighting was applied to the following functions, although it was fine for @ symbol. 
If I don't set a semantic color for functions, VSCode will fallback to the color of `entity.name.function`.
Current, `entity.name.function` scope color is blue.

This could be fixed by setting semantic highlighting to false, but this would affect other scopes as well.
